src/lib/y2storage/volume_specification.rb

```diff
@@ -408,6 +410,19 @@ def load_features(volume_features)
     # the architecture does not support to resume from swap (i.e., for s390).
     def adjust_features
       self.adjust_by_ram = false if swap? && !resume_supported?
+
+      return unless Y2Storage::BootRequirementsStrategies::BLS.bls_bootloader_proposed?
+
+      # Removing grub2/grub2-efi specific subvolumes because they are not needed.
+      @subvolumes.delete_if do |subvol|
```

Comment:
Why not to avoid adding these subvolumes in `SubvolSpecification.fallback_list` method?

Reply:
Hm, I am not sure if this is the right way. 
fallback_list will be used if there has not been defined a suggestion in the control.xml file:
https://github.com/yast/yast-storage-ng/blob/master/src/lib/y2storage/subvol_specification.rb#L239
There are grub2 defines subvolumes e.g. in the TW control.xml file. These will be currently dropped with this PR.
So, it is no problem to switch the default bootloader in the control.xml file e.g. back to grub2 without taking care about the subvolumes.
But again....I could be wrong

Reply:
I see. Probably the subvolumes related to booting should not defined by the control.xml file but by the boot strategy. So, each strategy would have a list of subvolumes, and then it adds the subvolumes if needed (similar to adding a separate mount point). But this is probably out of the scope of this PR. Could you please add a comment in that line? Explaining this option for the future.